### DISCUSSION TITLE
Suggestions from a theatre maker for minor improvements

### DISCUSSION THREAD

**Discussion by Lapazlane** [FrancescoCeruti/linux-show-player#325] [Ideas]
I'm using LSP for my show at the moment. Can I suggest the following for a future update:-
1.  Be able to save 'show index column' under Layout for an individual file, and/or have it as a default setting (with on/off toggle in preferences or layout menu).
2. Add a separate column for 'Cue Number' which would normally match the 'index column' number, but which allowed the possibility of  inserting new cues to an existing cue sheet that could be manually labelled with an alphabetic character, so that the following cue numbers (already written in the prompt book) stay the same. Eg  I want to add a new cue between existing Cues 24 and 25, so I add one which I label as cue '24a'
3. After completing the cue list, be able to LOCK the cue list with a toggle somewhere, to disable drag&drop cues - Iast week I had a sound operator who had not rehearsed the show or used my laptop (Thinkpad X260)  or LSP before and somehow managed, during the show to drag and drop one of the cues to a completely different location further down the list, without realising she had done so, messing up both that cue and the one after it.

**Comment by s0600204** [comment]
1. The "Show Index Column" option under the "Layout" menu is already saved into showfiles, but inverted (see #326)

2. Agreed, this would be useful. I currently prepend it to the cue name, but having a "proper" place to put it would be desirable. Would require more work, as there would need to be some check to ensure all the Cue Numbers were unique (within a given list). (Would also mean we wouldn't need the Index Column anymore, me thinks.)

4. Disable "Selection Mode" ("Layout" > "Selection Mode"; or keyboard shortcut Ctrl+Alt+S) and dragging of cues is not possible until it is re-enabled.

**Reply by FrancescoCeruti** [reply]
@Lapazlane Just to be sure, this are valid if you're using version 0.6 or higher
